osu.Game.Tests/Visual/SongSelectV2/TestSceneBeatmapTitleWedge.cs

```diff
@@ -115,6 +139,40 @@ public void TestWedgeVisibility()
             AddAssert("check visibility", () => titleWedge.Alpha > 0);
         }
 
+        [Test]
+        public void TestOnlineAvailability()
+        {
+            AddStep("online beatmapset", () =>
+            {
+                var (working, onlineSet) = createTestBeatmap();
+
+                currentOnlineSet = onlineSet;
+                Beatmap.Value = working;
+            });
+            AddAssert("play count = 10000", () => this.ChildrenOfType<BeatmapTitleWedge.Statistic>().ElementAt(0).Text.ToString() == "10,000");
+            AddAssert("favourites count = 2345", () => this.ChildrenOfType<BeatmapTitleWedge.Statistic>().ElementAt(1).Text.ToString() == "2,345");
+            AddStep("online beatmapset with local diff", () =>
+            {
+                var (working, onlineSet) = createTestBeatmap();
+
+                onlineSet.Beatmaps = Array.Empty<APIBeatmap>();
```

Comment:
Again with this. It makes zero sense to me, so please find a more correct way of doing this.

Reply:
https://github.com/ppy/osu/pull/32959/commits/19313ba8a423195337bcbd5d3a606d9ed7449c7d